ui/packages/consul-peerings/app/templates/dc/peers/index.hbs

```diff
@@ -126,12 +126,12 @@ as |sort filters items|}}
                 <BlockSlot @name="actions">
                   <li class="docs-link">
                     {{!-- what's the docs for peering?--}}
-                    <a href="https://www.consul.io/docs/agent/kv" rel="noopener noreferrer" target="_blank">
+                    <a href="{{env 'CONSUL_DOCS_URL'}}/connect/cluster-peering" rel="noopener noreferrer" target="_blank">
                       Documentation on Peers
                     </a>
                   </li>
                   <li class="learn-link">
-                    <a href="https://learn.hashicorp.com/consul/getting-started/kv" rel="noopener noreferrer" target="_blank">
+                    <a href="{{env "CONSUL_DOCS_URL"}}/connect/cluster-peering/create-manage-peering" rel="noopener noreferrer" target="_blank">
```

Comment:
```suggestion
                    <a href="https://www.consul.io/docs/connect/cluster-peering/create-manage-peering" rel="noopener noreferrer" target="_blank">
```

Reply:
Thanks for the suggestion but this is what the env - variable is adding. So no need to change this here.

Reply:
sounds good, thanks!